### PR TITLE
Removed usage of this.refs.errorMessage (Migrate string-refs to functional ones)

### PR DESCRIPTION
**Pull request by Raj-Datta-Manohar**
Fixes https://github.com/mattermost/mattermost-server/issues/15849

### CODE REVIEW

File: components/admin_console/schema_admin_settings.jsx [hunk 1]
```diff
@@ -86,6 +86,7 @@ export default class SchemaAdminSettings extends React.PureComponent {
             showConfirmId: '',
             clientWarning: '',
         };
+        this.errorMessage = React.createRef();
```
Comment (Willyfrog) [inline components/admin_console/schema_admin_settings.jsx:89]: for consistency with the rest of PRs can we rename this to:
```suggestion
        this.errorMessageRef = React.createRef();
```

**Review by Willyfrog — CHANGES_REQUESTED** [review]
looking good, added a request for consistency

**Review by Willyfrog — COMMENTED** [review]
looking good! I just noticed I forgot to mention to remove the comment that disables the eslint rule, can you remove it? is in line 3 and looks like:
```/* eslint-disable react/no-string-refs */```

**Comment by sudheerDev** [comment]
@Raj-Datta-Manohar Can you update the snapshots for tests?

**Review by sudheerDev — APPROVED** [review]
Thank you @Raj-Datta-Manohar 🎉

**Review by Willyfrog — APPROVED** [review]
LGTM! thanks for helping with this issue!

**Review by stevemudie — APPROVED** [review]
LGTM

**Comment by hanzei** [comment]
Thanks for helping us with improving the code quality :+1:

**Pull request merged**